documentation/modules/auxiliary/scanner/acpp/apple_acpp_scanner.md

```diff
@@ -0,0 +1,27 @@
+## Vulnerable Application
+
+ACPP is an undocumented and proprietary Apple protocol found in Airport products which protects the credentials used to administer the device. This module attempts exploit a weak encryption mechanism (fixed XOR key) by brute forcing the password via a dictionary attack or specific password.
+
+More information can be found on the [Rapid7 Vulnerability & Exploit Database page](https://www.rapid7.com/db/modules/auxiliary/scanner/acpp/login)
+
+## Verification Steps
+
+  1. Start msfconsole
+  2. Do: `use auxiliary/scanner/acpp/login`
+  3. Do: `show options`
+      ...show and set options...
+  4. Do: `run`
+
+## Scenarios
+
```

Comment:
Add an L3 header for what type of device (version, os) if possible.  For example:
```
### Apple Airport v2 OS 2.9
```